sledgehammer/start-up.sh

```diff
@@ -149,6 +149,11 @@ echo "# Sledgehammer added to log to the admin node" >> /etc/rsyslog.conf
 echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
+# Sometimes at this point network is not up yet, wait for it
+ping -c 1 -w 60 $ADMIN_IP > /dev/null || {
+    echo "error: $ADMIN_IP not reachable"
```

Comment:
If I recall correctly from the tests I made, the admin node just needed a bit more time to be reachable hence the 1 minute buffer time. The failure case is not dealt here. I can take a look at that and abort the script all together.